activestorage/lib/active_storage/attached/many.rb

```diff
@@ -44,20 +44,25 @@ def detach
       attachments.destroy_all if attached?
     end
 
+    ##
+    # :method purge
+    #
+    # :call-seq: User.highlights.purge
+    #
     # Directly purges each associated attachment (i.e. destroys the blobs and
     # attachments and deletes the files on the service).
-    def purge
-      if attached?
-        attachments.each(&:purge)
-        attachments.reload
-      end
-    end
+    #
+    # Attachments can be selectively purged using queries
+    # e.g. User.highlights.where(id: []).purge
 
-    # Purges each associated attachment through the queuing system.
-    def purge_later
-      if attached?
-        attachments.each(&:purge_later)
-      end
-    end
+    ##
+    # :method purge_later
```

Comment:
`:method: purge_later`

Reply:
Will format it like this in next push

Reply:
I think we should keep the documentation text. It is strange to only have the method name as documentation.